tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/KotlinLibSources.kt

```diff
@@ -115,15 +116,5 @@ fun kotlinComponent(module: String): ComponentIdentifier =
 
 
 private
-fun kotlinModuleVersionIdentifier(module: String): ModuleVersionIdentifier =
-    DefaultModuleVersionIdentifier("org.jetbrains.kotlin", module, embeddedKotlinVersion)
-
-
-private
-fun moduleId(group: String, module: String, version: String) =
-    object : ModuleComponentIdentifier {
-        override fun getGroup() = group
-        override fun getModule() = module
-        override fun getVersion() = version
-        override fun getDisplayName() = "$group:$module:$version"
-    }
+fun moduleId(group: String, module: String, version: String): ModuleComponentIdentifier =
+    DefaultModuleComponentIdentifier.newId(group, module, version)
```

Comment:
This is an internal class (and the interface is not meant for user extension either). Could you replace it with a predicate that checks the identifiers in `containsBuiltinKotlinModules`?

Reply:
That makes sense, @oehme, thanks.